pkg/goatfile/parser.go

```diff
@@ -810,6 +817,16 @@ func (t *Parser) parseFileDescriptor() (ast.DataContent, error) {
 	return fd, nil
 }
 
+func (t *Parser) parseRawDescriptor() (ast.DataContent, error) {
+	tk, varName := t.s.scanString()
+	if tk != tokSTRING {
+		return ast.NoContent{}, ErrInvalidFileDescriptor
+	}
+	rd := ast.RawDescriptor{VarName: varName}
+
+	return rd, nil
+}
```

Comment:
I think, it would make sense to also add the content type specification as same as with file descriptors here.

Example:
```
[FormData]
image = $uploadedImage:image/png

// or

[Body]
$uploadedImage:image/png
```

This should be relatively easy by using the same implementation from file descriptors.